### ISSUE TITLE
Support identi.ca

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#40]
```
Pino seems like a really cool Twitter client. It would be cool, if it would 
support identi.ca.

This is an enhancement request, but I can't find the option to mark it as 
such here. So please change as needed.
```

Original issue reported on code.google.com by `bigmike.MK` on 12 Jan 2010 at 5:01

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 12 Jan 2010 at 6:04
- Changed state: **Accepted**
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
I did a quick change locally and most things just work with changing the URLs 
to these:

protected const string friendsUrl = 
"http://identi.ca/api/statuses/home_timeline.xml";
protected const string statusUpdateUrl = 
"http://identi.ca/api/statuses/update.xml";
protected const string destroyStatusUrl = 
"http://identi.ca/api/statuses/destroy/%s.xml";
protected const string retweetStatusUrl = 
"http://identi.ca/api/statuses/retweet/%s.xml";
protected const string mentionsUrl = 
"http://identi.ca/api/statuses/mentions.xml";
protected const string userUrl = 
"http://identi.ca/api/statuses/public_timeline.xml";

There are a few glitches though, e.g. user icons don't seem to work correctly.

I tried to make this configurable through the preferences, but I failed and 
have no more time atm to do this. 
Sorry. :-)
```

Original comment by `bigmike.MK` on 13 Jan 2010 at 2:13

**Comment by GoogleCodeExporter** [comment]
```
Little update for userUrl
private const string userUrl = "http://identi.ca/api/users/show/%s.xml";

still no avatars :(
```

Original comment by `smarg...@gmail.com` on 30 Jan 2010 at 6:33

**Comment by GoogleCodeExporter** [comment]
```
Please, be patient, I'm already working on support of identi.ca. It's not so 
easy, as 
just to replace urls.
```

Original comment by `tro...@gmail.com` on 30 Jan 2010 at 6:39

**Comment by GoogleCodeExporter** [comment]
```
Hi troorl,

I'm patient, I just added an info. I thought it was the proper place.
But if you're already working on it, I will stop investigating ;)
```

Original comment by `smarg...@gmail.com` on 30 Jan 2010 at 6:58

**Comment by GoogleCodeExporter** [comment]
```
Closed by 011c9e4779 revision.
```

Original comment by `tro...@gmail.com` on 17 Feb 2010 at 7:23
- Changed state: **Fixed**
